tests/services/test_text_processor.py

```
@@ -161,22 +147,22 @@ def test_context_manager(self) -> None:
-            with patch("services.text_processor.ensure_nltk_data"):
-                with pytest.raises(RuntimeError, match="boom"):
-                    with TextProcessor() as _processor:
-                        raise RuntimeError("boom")
-                mock_model.safe_cleanup.assert_called_once()
+        with (
+            patch("services.text_processor.get_text_model", return_value=mock_model),
+            pytest.raises(RuntimeError, match="boom"),
+        ):
+            with TextProcessor() as _processor:
+                raise RuntimeError("boom")
+            mock_model.safe_cleanup.assert_called_once()
```

Ruff (0.15.10)

[warning] `pytest.raises()` block should contain a single simple statement

(PT012)